### PR TITLE
feat: [STO-6146]: Add SBOM generation support for blackduck

### PR DESCRIPTION
**Pull request by williamwissemann**
### Summary

Screenshot 2023-07-12 at 7 06 27 PM

Screenshot 2023-07-12 at 7 10 10 PM

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by williamwissemann** [comment]
retrigger titlecheck

**Comment by williamwissemann** [comment]
retrigger titlecheck

File: src/modules/85-sto/components/PipelineSteps/SecurityFields.tsx [hunk 1]
```diff
@@ -32,13 +34,16 @@ import {
...
   inputSetInstanceFields,
+  inputSetSbomFields,
   inputSetScanFields,
   inputSetTargetFields,
   inputSetToolFields,
   instanceProtocolSelectItems,
   JFROG_ARTIFACTORY_CONTAINER_TYPE,
   LOCAL_IMAGE_CONTAINER_TYPE,
   logLevelOptions,
+  SBOM_CYCLONE_DX,
```
Comment (deepharness) [inline src/modules/85-sto/components/PipelineSteps/SecurityFields.tsx:45]: CycloneDx is single word

Comment (williamwissemann) [reply src/modules/85-sto/components/PipelineSteps/SecurityFields.tsx]: Fixed

**Review by jsentongo — APPROVED** [review]

**Review by deepharness — APPROVED** [review]

**Pull request merged**